include/libdnf/rpm/reldep.hpp

```diff
@@ -47,16 +47,13 @@ class Reldep {
     /// @replaces libdnf/repo/solvable/Dependency.hpp:method:Dependency(const Dependency & dependency);
     Reldep(const Reldep & reldep);
 
-    Reldep(const Reldep && reldep) noexcept;
-
     /// @replaces libdnf/repo/solvable/Dependency.hpp:method:~Dependency();
     /// @replaces libdnf/dnf-reldep.h:function:dnf_reldep_free(DnfReldep *reldep)
     ~Reldep();
 
     bool operator==(const Reldep & other) const noexcept;
     bool operator!=(const Reldep & other) const noexcept;
     Reldep & operator=(const Reldep & other) noexcept = default;
-    Reldep & operator=(Reldep && other) noexcept;
```

Comment:
I believe that the method was added, because clang complained that the method is not there :)

Reply:
Ok. I changed it. I use explicitly `delete`.
I cleaned up the code and used the agreed style. So, the "clang-tidy" does not complain about Reldep.